Content.Shared/Chat/ChatModifiers/MainMessageChatModifier.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Robust.Shared.Utility;
+
+namespace Content.Shared.Chat.ChatModifiers;
+
+/// <summary>
+/// Wraps a [MainMessage] tag around all text nodes, which can later be utilized by other chat modifiers.
```

Comment:
maybe elaborate what actually main message is? it would be a good place for ppl who are trying to find out how all this stuff works :3

Reply:
other then the actual docs, ofc